arm-eventhub/2015-08-01/swagger/EventHub.json

```diff
@@ -41,7 +39,7 @@
         "tags": [
           "Namespaces"
```

Comment:
Does eventhub RP supports API to check the availability of namespace? if yes can we add it?